### PR TITLE
Use create-react-app for react boilerplate

### PR DESCRIPTION
**Pull request by velcrin**
Using `create-react-app` to kick start a react project enable us to avoid having to update the boilerplate every time a new version of react is released. This is the market defacto tool to get started on a react project. Also, it contains less dependencies than the current boilerplate making the project simpler to get started with and letting more freedom over the tools that the project actually needs.

The `init.sh` seems to work but I am not sure how to test the full process. Any help appreciated.

### CODE REVIEW

**Comment by gerardcl** [comment]
nice @velcrin ! have you tested deploying it from the prov app?

File: boilerplates/fe-react/README.md [hunk 1]
```diff
@@ -2,79 +2,30 @@
...
-
- This quickstarter generates a React project for developing webpages.
-
- It contains the basic setup for [Docker](https://www.docker.com/), [Jenkins](https://jenkins.io/), [SonarQube](https://www.sonarqube.org/) and [OpenShift](https://www.openshift.com/), so you have your CI/CD process out of the box.
-
-
- ## What files / architecture is generated?
-
- ```
-├── Jenkinsfile - This file contains Jenkins build configuration settings
```
Comment (m-apsolon) [inline boilerplates/fe-react/README.md]: if the file structure changed, update, but don't move the files structure

Comment (velcrin) [reply boilerplates/fe-react/README.md]: I added a direct link to create-react-app file structure

**Comment by velcrin** [comment]
@gerardcl I setted up a meeting to look at this together

**Comment by gerardcl** [comment]
@m-apsolon we close it since the right one now is coming from bix-digital fork, see: https://github.com/opendevstack/ods-project-quickstarters/pull/228

**Pull request closed without merging**